JPS_Ontology/ontology/ontosealevel/OntoSeaLevel.csv

```diff
@@ -0,0 +1,34 @@
+Tree,Class,CulturalSite,IS-A,,,,Tree,https://www.theworldavatar.com/kg/ontosealevel/,Tree
+HeritageTree,Class,Tree,IS-A,,,,A tree that is the natural heritage of a specific place.,https://www.theworldavatar.com/kg/ontosealevel/,HeritageTree
+https://schema.org/Museum,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Museum
+https://schema.org/TouristAttraction,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Tourist attraction
+https://schema.org/LandmarksOrHistoricalBuildings,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Landmarks or historical building
+Monument,Class,CulturalSite,IS-A,,,,A structure for the commemoration of a person or event.,https://www.theworldavatar.com/kg/ontosealevel/,Monument
+Monument,Class,https://raw.githubusercontent.com/beAWARE-project/ontology/master/beAWARE_ontology#Monument,EQUIVALENT-TO,,,,A structure for the commemoration of a person or event.,https://www.theworldavatar.com/kg/ontosealevel/,Monument
+Scenario,Class,,,,,,Scenario that can act as input to a forecasting level for forecasting sea level.,https://www.theworldavatar.com/kg/ontosealevel/,Scenario
+SharedSocioEconomicPathway,Class,Scenario,IS-A,,,,A climate change scenario of projected socioeconomic changes.,https://www.theworldavatar.com/kg/ontosealevel/,Shared Socioeconomic Pathway
+measuredAt,Object Property,,,LocalMeanSeaLevel,http://www.opengis.net/ont/geosparql#Feature,exactly 1,A relationship between the local mean sea level and the reference point for the local mean sea level measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured at
+measuredIn,Data Property,,,SeaLevelMeasurement,http://www.w3.org/2001/XMLSchema#gYear,,A relationship between a sea level measurement and the time of measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured in
+measuredIn,Data Property,,,SeaLevelMeasurement,http://www.w3.org/2001/XMLSchema#date,,A relationship between a sea level measurement and the time of measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured in
+hasHeight,Object Property,,,SeaLevelMeasurement,http://www.ontology-of-units-of-measure.org/resource/om-2/Height,,A relationship between a sea level measurement and the sea level height.,https://www.theworldavatar.com/kg/ontosealevel/,has height
+isAffectedBy,Object Property,,,https://www.theworldavatar.com/kg/ontobuiltenv/Property,EconomicDamage,,A relationship between the damage due to rising sea level and the places it affects.,https://www.theworldavatar.com/kg/ontosealevel/,is affected by
+isAffectedBy,Object Property,,,CulturalSite,CulturalDamage,,A relationship between the damage due to rising sea level and the places it affects.,https://www.theworldavatar.com/kg/ontosealevel/,is affected by
```

Comment:
Similar to the comment on `measuredIn`, I believe that in order to make this correct, you need one statement defining `isAffectedBy` with both the domain and the range being the union of the relevant concepts. In addition, you then need statements that restrict values (using the `only` qualifier), so that one cannot make statements like `CulturalSite3 isAffectedBy EconomicDamage5`.